modules/custom/az_migration/src/Plugin/migrate/process/ParagraphsUpdateMediaCaption.php

```diff
@@ -0,0 +1,38 @@
+<?php
+
+namespace Drupal\az_migration\Plugin\migrate\process;
+
+use Drupal\migrate\MigrateExecutableInterface;
+use Drupal\migrate\ProcessPluginBase;
+use Drupal\migrate\Row;
+use Drupal\media\Entity\Media;
+
+/**
+ * Configure Behavior for paragraphs.
```

Comment:
This comment needs updating to reflect what this process plugin does.

Reply:
Done